Software_Carbon_Intensity/Appendix_A_Reporting_Requirement_Examples.md

```diff
@@ -0,0 +1,148 @@
+# Appendix A: Reporting Requirement Examples
+This appendix provides information of valid machine readable reporting files in JSON, and YAML. Other machine readable file formats are acceptable, so long as they can be parsed in a way that allows for the same identifiers to be extracted.
```

Comment:
I think this is too confusing, we have to stick to YAML (or JSON). If you want to store in JSON in your system and convert to YAML that's fine but by the time it comes to reporting it has to be one format. Otherwise we have a world of pain in our future trying to deal with the edge cases from having multiple formats and how anyone wanting to parse our database will need to deal with multiple formats.